core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java

```diff
@@ -108,6 +108,9 @@
 import io.trino.spi.statistics.ComputedStatistics;
 import io.trino.spi.statistics.TableStatistics;
 import io.trino.spi.statistics.TableStatisticsMetadata;
```

Comment:
pls add to the description of the PR the concrete issue that motivated you to create the PR

```
CREATE TABLE table (x timestamp); → success
CREATE TABLE table (x array(timestamp)); → Timestamp precision (3) not supported for Iceberg. Use "timestamp(6)" instead.
```